package.json

```diff
@@ -1,22 +1,30 @@
 {
-	"name": "@kibertoad/biome-config",
-	"version": "1.2.1",
-	"description": "",
-	"type": "module",
-	"scripts": {
-		"format": "biome check --write ."
-	},
-	"repository": {
-		"type": "git",
-		"url": "https://github.com/kibertoad/biome-config.git"
-	},
-	"files": ["configs/biome-package.json", "LICENSE", "README.md"],
-	"homepage": "https://github.com/kibertoad/biome-config",
-	"exports": {
-		"./*": ["./configs/*.json"]
-	},
-	"private": false,
-	"devDependencies": {
-		"@biomejs/biome": "^1.8.2"
-	}
+  "name": "@kibertoad/biome-config",
+  "version": "1.2.1",
+  "description": "",
+  "type": "module",
+  "private": false,
+  "scripts": {
+    "format": "biome check .",
+    "format:fix": "biome check . --write"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/kibertoad/biome-config.git"
+  },
+  "homepage": "https://github.com/kibertoad/biome-config",
+  "files": [
+    "configs/biome-package.json"
+  ],
+  "exports": {
+    "./*": [
+      "./configs/*.json"
+    ]
+  },
```

Comment:
I removed `LICENSE` and `README.md` from `files` since they are included by default in `npm pack`

<img width="675" alt="Screenshot 2025-06-27 at 05 21 36" src="https://github.com/user-attachments/assets/33b8566a-38c1-4c1e-b541-c3cfae4972c7" />